test/steps/debug_steps.go

```diff
@@ -0,0 +1,19 @@
+package steps
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+
+	"github.com/cucumber/godog"
+)
+
+// DebugSteps defines Gherkin step implementations around merge conflicts.
```

Comment:
This comment seems wrong

Reply:
Good catch, fixed in https://github.com/git-town/git-town/pull/1355